content/en/platform/corda/4.3/enterprise/corda-network/UAT.md

```diff
@@ -48,4 +48,4 @@ Corda Network Pre-Production is governed by an [independent Foundation](https://
 
 Steps to join are outlined on the [Corda Network microsite](https://corda.network/participation/index.html) - follow any specific instructions for ‘Pre-Production’.
```

Comment:
Apologies if I missed this in the earlier review - this link doesn't work and should be replaced with
https://corda.network/joining-corda-network/onboarding-workflow

Reply:
Changed throughout